streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -476,7 +476,67 @@ public void shouldNotGetTaskWithKeyAndPartitionerWhenNotRunning() {
         globalStreams.metadataForKey("store", "key", (topic, key, value, numPartitions) -> 0);
     }
 
+    static class MyStringSerializer extends StringSerializer {
+        boolean configured = false;
+        boolean called = false;
+
+        @Override
+        public void configure(final Map<String, ?> configs, final boolean isKey) {
+            super.configure(configs, isKey);
+            configured = true;
+        }
+
+        @Override
+        public byte[] serialize(final String topic, final String data) {
+            called = true;
+            return super.serialize(topic, data);
+        }
+
+        boolean configured() {
+            return !called || configured;
+        }
+    }
+
+    static class MyStringDeserializer extends StringDeserializer {
+        boolean configured = false;
+        boolean called = false;
+
+        @Override
+        public void configure(final Map<String, ?> configs, final boolean isKey) {
+            super.configure(configs, isKey);
+            configured = true;
+        }
+
+        @Override
+        public String deserialize(final String topic, final byte[] data) {
+            called = true;
+            return super.deserialize(topic, data);
+        }
+
+        boolean configured() {
+            return !called || configured;
+        }
+    }
+
+    public static class MyStringSerde<K> extends Serdes.WrapperSerde<String> {
+        public MyStringSerde() {
+            super(new MyStringSerializer(), new MyStringDeserializer());
+        }
+
+        @Override
+        public void configure(final Map<String, ?> configs, final boolean isKey) {
+            super.configure(configs, isKey);
+        }
+
+        public boolean configured() {
+            if (!((MyStringSerializer) this.serializer()).configured()) return false;
```

Comment:
This can be simplified to 
```java
return ((MyStringSerializer) serializer()).configured() &&
                ((MyStringDeserializer) deserializer()).configured();
```